Kitodo/src/main/webapp/WEB-INF/templates/includes/desktop/processesWidget.xhtml

```diff
@@ -20,7 +20,6 @@
                  value="#{DesktopForm.processes}"
                  scrollable="true"
                  styleClass="default-layout"
-                 sortBy="#{process.id}"
```

Comment:
If you remove `sortBy`, you should probably also remove the following line, because `sortOrder` doesn't make sense without `sortBy`.